src/sidebar/components/test/accessibility.js

```diff
@@ -0,0 +1,87 @@
+import { run } from 'axe-core';
+import { ReactWrapper, mount } from 'enzyme';
+import { isValidElement } from 'preact';
+
+/**
+ * @typedef {Scenario}
+ * @prop {string} [name] -
+ *   A descriptive name for the scenario. Defaults to "default" if not specified.
+ * @prop {() => import('preact').VNode|ReactWrapper} content -
```

Comment:
We use the flavor of JSDoc as understood by TypeScript, which [is documented here](https://www.typescriptlang.org/docs/handbook/type-checking-javascript-files.html#supported-jsdoc). The reason for that is because it is well supported by tooling and IDEs, such as VS Code.

In that variant, `import(<module path>).<type name>` is a way of referring to types that are defined in another module or package. This works both for our own modules and third-party packages.

The original definition of the type could be ordinary JavaScript (eg. `class Foo`), a JSDoc (`@typedef`) or a `.d.ts` TypeScript definition file. In this case the "preact" module ships with an [`index.d.ts` file](https://github.com/preactjs/preact/blob/383443976e418033e448505aa6b4c53a07a65be4/src/index.d.ts#L13) which defines a `VNode` type. VNodes ("virtual DOM nodes") are what `createElement` (and hence, JSX expressions) return.

Reply:
okay, makes sense, thanks